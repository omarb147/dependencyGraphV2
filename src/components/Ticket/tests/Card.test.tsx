import React from 'react';
import 'jest-styled-components';
import { render } from '@testing-library/react';
import Ticket from '..';

describe('<Ticket/>', () => {
  it('should render a Ticket with color and text', () => {
    const { container } = render(<Ticket itemId="123-321" selected={false} />);
    expect(container).toMatchSnapshot();
  });
});
