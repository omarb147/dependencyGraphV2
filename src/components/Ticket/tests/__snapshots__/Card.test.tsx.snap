// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Ticket/> should render a Ticket with color and text 1`] = `
.c0 {
  background-color: #ffffff;
  border: none;
  border-top: 5px solid yellow;
  width: 100%;
  box-shadow: 0px 5px 10px #c9c9c9;
  padding: 0.75rem;
}

.c1 {
  color: #434343;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
}

.c2 {
  padding: 0.25rem 0.4rem;
  font-size: 0.8rem;
  background-color: yellow;
  color: #ffffff;
  margin-right: 0.5rem;
  opacity: 0.7;
}

.c4 {
  margin: 0.5rem 0 0.25rem;
  border-top: 1px solid #e1e1e1;
}

.c3 {
  font-size: 0.9rem;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  color: #a5a5a5;
  font-size: 0.7rem;
}

<div>
  <div
    class="c0"
    color="yellow"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
        color="yellow"
      >
        2
      </div>
      <div
        class="c3"
      >
        Hello World
      </div>
    </div>
    <hr
      class="c4"
    />
    <div
      class="c5"
    >
      label
    </div>
  </div>
</div>
`;
